cmd/neofs-cli/modules/container/delete.go

```diff
@@ -104,17 +106,21 @@ Only owner of the container has a permission to remove container.`,
 			getPrm.SetClient(cli)
 			getPrm.SetContainer(id)
 
-			for i := 0; i < awaitTimeout; i++ {
+			for {
 				time.Sleep(1 * time.Second)
 
-				_, err := internalclient.GetContainer(cmd.Context(), getPrm)
+				select {
+				case <-ctx.Done():
+					common.ExitOnErr(cmd, "", errDeleteTimeout)
```

Comment:
Also, these "waited for too long" cases should have a special exit code

Reply:
Consider rephrasing `errDeleteTimeout` (and alike) as well to say that "the operation was requested, but timeout has happened while waiting for its result".

Reply:
> Also, these "waited for too long" cases should have a special exit code

overall i agree, but think it's better to do within separate issue since previously there were no dedicated codes for these cases

Reply:
> Consider rephrasing

seems good, done

Reply:
#2380.